explainer.md

```diff
@@ -98,7 +98,16 @@ An interesting wrinkle is how such a SW would get bootstrapped. We discuss one o
 
 ## Mommy, Where Do Service Workers Come From?
 
-Since both connection types require the target service worker to actually be installed before communication is possible, a new way of installing a service worker is needed. To enable this we introduce two new headers as part of the http response for any resource. Whenever these headers are present, the browser will install a service worker in the background (not blocking the original request), allowing future communication with the new service worker.
+Since both connection types require the target service worker to actually be installed before communication is possible, a new way of installing a service worker is needed. To enable this we introduce two new headers as part of the http response for any resource:
+
+```
+Service-Worker-Scope: scopeURL
+Service-Worker-Script: scriptURL
+```
+
+If the `Service-Worker-Scope` header is absent, the maximum scope allowed is the script's location. Otherwise, the maximum scope is what the `Service-Worker-Allowed` header suggests.
```

Comment:
Should it be made clear that the SWA header is taken from the request to scriptURL, which is a different request from where the SW-Scope and SW-Script headers were found? Although the example might make that clear enough.
Also the presence or absence of the SW-Scope header does not really change the maximum scope I'd imagine. If the header is absent the scope will just default to the script's location.